net/zerotier/patches/0005-do-not-use-zt-SSO-and-OIDC.patch

```diff
@@ -0,0 +1,35 @@
+From c368d51cb1d221cf3ffb2d33d03dcc69591339c0 Mon Sep 17 00:00:00 2001
+From: Moritz Warning <moritzwarning@web.de>
+Date: Sun, 13 Mar 2022 10:06:08 +0100
+Subject: [PATCH 5/5] do not use zt SSO and OIDC
```

Comment:
My usual questions, which applies to patches:

1. Can you please add commit message, why this is necessary?
2. It is possible to upstream this patch?

Reply:
1. done
2. very unlikely, we use the fall back solution they provided, since the OpenWrt build system does not support Rust.